app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class PostsController < Api::V1::BaseController
+      include PostsHelper
+
+      before_action only: :show do
+        require_access_token %w[read]
+      end
+
+      before_action only: %i[create destroy] do
+        require_access_token %w[read write]
+      end
+
+      def show
+        posts_services = PostService.new(id: params[:id], user: current_user)
+        posts_services.mark_user_notifications unless params[:mark_notifications] == "false"
```

Comment:
The `#mark_user_notifications` method requires exactly one argument. Since there are no failing tests you probably don't have a spec for the case when `params[:mark_notifications]` is not `false`. So I guess you should add this spec.

Reply:
Thx for the review, but I didn't have clean that one yet. I will handle it when I will do it.